apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/FirestoreMultipleDbTest.cs

```diff
@@ -16,29 +16,27 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Xunit;
-using static Google.Cloud.Firestore.Admin.V1.Database.Types;
 
 namespace Google.Cloud.Firestore.IntegrationTests;
 
-[Collection(nameof(MultipleFirestoreDbFixture))]
+[Collection(nameof(FirestoreFixture))]
 public class FirestoreMultipleDbTest
 {
-    private readonly MultipleFirestoreDbFixture _fixture;
-    public FirestoreMultipleDbTest(MultipleFirestoreDbFixture multipleFirestoreDbFixture) =>
-        _fixture = multipleFirestoreDbFixture;
+    private readonly FirestoreFixture _fixture;
+    public FirestoreMultipleDbTest(FirestoreFixture fixture) =>
+        _fixture = fixture;
 
     [Fact(Skip = "This is a preview feature and it's not yet stable.")]
```

Comment:
Unskipp maybe, if it was flaky because of lack of polling, it should work now.

Reply:
Whoops, done - that was definitely the intention.